Polyhedron/demo/Polyhedron/include/CGAL/IO/read_surf_trianglemesh.h

```diff
@@ -0,0 +1,186 @@
+#include <iostream>
+#include <string>
+#include <CGAL/boost/graph/Euler_operations.h>
+#include <CGAL/Polygon_mesh_processing/internal/named_function_params.h>
+#include <CGAL/Polygon_mesh_processing/internal/named_params_helper.h>
+
+//a material is composed of an material Id and a material name.
+typedef std::pair<int, std::string> material;
+
+struct MaterialData
+{
+  material innerRegion;
+  material outerRegion;
+};
+
+/*!
+ * read_surf reads a file which extension is .surf and fills `output` with one `Mesh` per patch.
+ * Mesh is a model of FaceListGraph.
+ */
+template<class Mesh, class NamedParameters>
+void read_surf(std::istream& input, std::vector<Mesh>& output, std::vector<MaterialData>& metadata,  const NamedParameters&)
+{
+  typedef typename GetGeomTraits<Mesh,
```

Comment:
in `master`, `GetGeomTraits` now is `CGAL::GetGeomTraits`.
It should be updated before testing.
Thanks!